### DISCUSSION TITLE
ValueError: No chrome executable found on PATH

### DISCUSSION THREAD

**Discussion by HHreal** [omkarcloud/google-maps-scraper#53] [General]
Whenever i try to start scraping it says: ValueError: No chrome executable found on PATH

**Comment by Chetan11-dev** [comment]
This means you do not have Chrome Installed on your System which is needed to run this scraper.
Alternatively, you can run it in Gitpod, a cloud-based browser IDE, as follows:
1. Open Repository in Gitpod by visiting [here](https://gitpod.io/#https://github.com/omkarcloud/google-maps-scraper) and signing up with GitHub.
   ![Screenshot (148)](https://github.com/omkarcloud/google-maps-scraper/assets/53407137/f498dda8-5352-4f7a-9d70-c717859670d4)

2. In the terminal, run `docker-compose build && docker-compose up`.

3. Find the leads in the `output` folder.

**Reply by HHreal** [reply]
Thank you for helping. GitHub works perfectly.

**Comment by Charlotte-br560** [comment]
Make sure you have Chrome installed and its executable is added to your system's PATH. If Chrome is already installed, you might need to specify the path to the Chrome executable in your code or update your system's PATH variable.
